var/spack/repos/builtin/packages/esmf/package.py

```diff
@@ -18,7 +18,7 @@ class Esmf(MakefilePackage):
     homepage = "https://www.earthsystemcog.org/projects/esmf/"
     url = 'https://github.com/esmf-org/esmf/archive/ESMF_8_0_1.tar.gz'
 
-    version('8.1.1',  sha256='58c2e739356f21a1b32673aa17a713d3c4af9d45d572f4ba9168c357d586dc75')
+    version('8_1_1',  sha256='58c2e739356f21a1b32673aa17a713d3c4af9d45d572f4ba9168c357d586dc75')
```

Comment:
@kgerheiser what is the deal with this change of 8.1.1 to 8_1_1?

Oddly, it works either way! The only difference is the name of the directory it gets installed in.

However, seems like 8.1.1 is the more natural choice, and the one that is already in use, so why try to change it to 8_1_1?

Reply:
I think I updated to 8_1_1 (or vice-versa) and in the meantime someone added it to the Spack repo and they were both merged.

This will have to be changed to use Git so we can install beta release tags.

Reply:
OK, I changed it to 8.1.1 for consistency.